### DISCUSSION TITLE
Swtich Critical Error

### DISCUSSION THREAD

**Discussion by iAmNotEric** [jordanbtucker/totk-no-loot-boxes#5] [Q&A] [answered]
Hey, if i installed your Mod my Switch will crash when ill launche Totk.
Im on 1.1.0 and my Atmosphere is on 1.5.4 / NintendoOS 16.0.3.

Path is /atmosphere/contents/0100F2C0115B6000/romfs

What i'm doing wrong?

**Comment by jordanbtucker** [comment, marked as answer]
A few things:
- The mod only contains one file. Can you make sure that file is at `/atmosphere/contents/0100F2C0115B6000/romfs/Pack/ResidentCommon.pack.zs`?
- What version of the mod are you using: v1.0.0 or v1.1.0?
- What error messages, if any, do you get when the game crashes?

**Reply by iAmNotEric** [reply]
Oh man, i'm so stupid....
Forget what I wrote.I  had Atmo 1.5.3 instead of 1.5.4.

Especially it is noted EXTRA in the requirements!
"running Atmosphere 1.5.4 or later".

The Mod running now as requested. :D


Sorry that I have unnecessarily wasted your time

**Comment by PraiseThyJeebus** [comment]
TotK 1.1.1
On switch
No other mods
On atmosphere 1.5.4
NoLootBoxes v2.0.1
With the mod placed in "Nintendo Switch\SD Card\atmosphere\contents\0100F2C0115B6000\romfs" I get a error about a panic occurring while running atmosphere when launching TotK
I have tried removing the mod, the game runs fine, but without the mod.
I have verified I am on 1.5.4 by reinstalling it.

I have obviously done something wrong at some point, but I do not know what

**Reply by jordanbtucker** [reply]
Sorry for the late reply. I'm at GDQ this week. What error code are you getting?

**Reply by Growthor** [reply]
I'm have almost the same specifications.

TotK 1.0.0
On switch
No other mods
On atmosphere 1.5.4
NoLootBoxes v2.0.1

SD Card\atmosphere\contents\0100F2C0115B6000\romfs\Pack\ResidentCommon.pack.zs

The error simply says "The software was closed because an error occurred."

I've tested other mods, but I'm not using anything simultaneously. And even still, no other mod I tested uses ResidentCommon.pack.zs
Maybe the switch needs more files?

Let me know if I can help testing somehow.

**Reply by PraiseThyJeebus** [reply]
> Sorry for the late reply. I'm at GDQ this week. What error code are you getting?

I actually got help in the TotK modding discord yesterday morning, Turned out sys-botbase was causing problems
